code/crates/core-types/src/threshold.rs

```diff
@@ -79,7 +79,7 @@ impl ThresholdParam {
 
     /// Return the minimum expected weight to meet the threshold when applied to the given total.
     pub fn min_expected(&self, total: VotingPower) -> VotingPower {
-        total
+        1 + total
```

Comment:
👍 Maybe we can also fix the log [here](https://github.com/informalsystems/malachite/blob/eb0cf85a94069a2a6cd34e762df3b699f5bcaf08/code/crates/core-consensus/src/state.rs#L234)